cmd/nomoserrors/examples/examples.go

```diff
@@ -321,6 +321,12 @@ func Generate() AllExamples {
 	// 1069
 	result.add(validate.SelfReconcileError(fake.RootSyncV1Beta1(configsync.RootSyncName)))
 
+	// 1070
+	result.add(selectors.UnsupportedNamespaceSelectorModeError(fake.NamespaceSelector()))
+
+	// 1071
+	result.add(selectors.UnknownNamespaceSelectorModeError(fake.NamespaceSelector()))
```

Comment:
Is it necessary to separate unsupported vs. unknown? The error can just be "unsupported / invalid" to let users know that the specification is not valid.

Reply:
`unknown` applies to both hierarchy mode and unstructured mode, while `unsupported` only applies the hierarchy mode.

We have KNV1014 for invalidSelector, which applies to both ClusterSelector and NamespaceSelector: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/importer/analyzer/transform/selectors/errors.go#L29. There is also a TODO to refactor it properly. 

What's the best practice to reuse error code? Do we need to group them for a particular feature?

Reply:
Please see my other comment

Reply:
changed to reuse KNV1014